.github/workflows/release.yml

```diff
@@ -18,8 +18,24 @@ jobs:
       - uses: actions/checkout@v4
 
       - uses: prefecthq/actions-release-ui-components@main
+        id: release-ui-components
         with:
           NPM_TOKEN: ${{ secrets.PREFECT_UI_COMPONENTS_NPM_TOKEN }}
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          NEBULA_UI_ACTIONS_RW_TOKEN: ${{ secrets.NEBULA_UI_ACTIONS_RW }}
-          PREFECT_OSS_ACTIONS_RW_TOKEN: ${{ secrets.PREFECT_ACTIONS_RW }}
+          SKIP_BUILD: 'true'
+
+      - uses: prefecthq/actions-trigger-downstream-npm-package-updates@main
+        id: trigger-downstream-npm-package-update-nebula-ui
+        if: ${{ steps.release-ui-components.publish.outputs.type }}
+        with:
+          GITHUB_TOKEN: ${{ secrets.NEBULA_UI_ACTIONS_RW }}
+          DOWNSTREAM_REPO_NAME: nebula-ui
+          RELEASE_TAG: ${{ steps.release-ui-components.publish.outputs.version}}
+
+      - uses: prefecthq/actions-trigger-downstream-npm-package-updates@main
+        id: trigger-downstream-npm-package-update-prefect-oss
+        if: ${{ steps.release-ui-components.publish.outputs.type }}
+        with:
+          GITHUB_TOKEN: ${{ secrets.PREFECT_ACTIONS_RW }}
+          DOWNSTREAM_REPO_NAME: prefect
+          RELEASE_TAG: ${{ steps.release-ui-components.publish.outputs.version}}
```

Comment:
Is this action at a point that we could plop in _all_ of the other downstream packages (pretty much all of our npm packages use this) so
- prefect-design
- prefect-ui-library
- graphs
- vue-compositions

or do we also need separate secrets for each still?

(😅 - i'd like to cut a new version of this "soon". suuuuuper no rush tho. "soon" could be really whenever, but it'd be neat to test all of this out with the next version bump)

Reply:
We would need to create GH secrets still but I can get that done today/monday!

Reply:
that would be dope! 

will save me _so_ many button clicks!!! 

<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExZjA5ZnZtZmNlYnl5OWxndjE5Z2J6bHp1MDlxM3N2Z2lkYWhmeHB0dCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/oF5oUYTOhvFnO/giphy.gif" />